autometa/common/coverage.py

```diff
@@ -239,4 +219,24 @@ def main(args):
         default=False)
     parser.add_argument('--out', help='</path/to/coverages.tsv>', required=True)
```

Comment:
For this and other help messages (I couldn't get github to select the grey parts), need to explain more simply what the files are that the script is expecting. So rather than </path/to/coverages.tsv>, write "Path to write a table of coverages".